### ISSUE TITLE
[BUG] Windows setup script fails

### ISSUE THREAD

**Issue by t0mpr1c3** [AllYarnsAreBeautiful/ayab-desktop#618]
**Environment**
AYAB software version:  1.0.0-dev
Computer/OS:  Windows 11
Knitting machine:  -
AYAB hardware:  -

**Describe the bug**
Conda install script fails with error message 
```
ERROR: Could not find a version that satisfies the requirement PyQt5==5.12.3 (from versions: 5.14.0, 5.14.1, 5.14.2, 5.15.0, 5.15.1, 5.15.2, 5.15.3, 5.15.4, 5.15.5, 5.15.6, 5.15.7, 5.15.8, 5.15.9, 5.15.10)
ERROR: No matching distribution found for PyQt5==5.12.3
```

**To Reproduce**
Open Anaconda powershell prompt. Attempt to run Windows install script in `README.md`

![image](https://github.com/AllYarnsAreBeautiful/ayab-desktop/assets/1699266/63d383e5-163b-4926-97b3-53c6290889c5)

**Comment by t0mpr1c3** [comment]
Suggested fix is to bump PyQt5 version to 5.15.10

**Comment by t0mpr1c3** [comment]
Oops, it looks like I had not activated the `ayab` environment. Closing this issue, although the suggested version bump does seem to work.
